frontend/components/TableContainer/DataTable/TruncatedTextCell/_styles.scss

```diff
@@ -10,5 +10,32 @@
   .truncated {
     display: inline-block;
     max-width: 100%;
+
+    // Removes secondary unstyled tooltip created by browser
+    &::after {
+      content: "";
+      display: block;
+    }
+  }
+}
+
+// Responsive design overrides react-tooltip width that overflows off screen
+.tooltip {
```

Comment:
Should this be inside of `.data-table__truncated-text`? It looks like this selector will apply to all instances of `.tooltip` through the app.

Reply:
omg good catch, will ping you for a re-review